dellmont-credit-checker.sh

```diff
@@ -202,7 +202,9 @@ check_credit_level() {
 				local PREVCREDIT=("2000-01-01 00:00 0")
 			fi
 			echo -e "`date +"%Y-%m-%d %T"`\t$CREDITCENTS">>"$6"
-			local CREDITFALL=$((${PREVCREDIT[2]}-$CREDITCENTS))
+			# remove leading spaces if any, and add 10# so to make it work with credit like: "093" " 201"
+			temp_var=`echo $CREDITCENTS | sed -e 's/^[ \t]*//'`
```

Comment:
Why do you use a temporary variable here ?
Wouldn't it be cleaner to nest the command substitution into the next line ?